docs/en/modules/admin/pages/participants/verifications_conflicts.adoc

```diff
@@ -1,38 +1,60 @@
 
 For instance:
 
-. A participant get xref:admin:participants/impersonations.adoc[impersonated], and they don't provide an email
-. Then they create an account and try to verify themselves with the Census verification
+. A participant get xref:admin:participants/impersonations.adoc[impersonated] with their data, but don't enter an email
+. Then they create an account, and try to verify themselves with the same verification
 
-For security, we try to be extra cautious in these cases, and we want that an administrator reviews these verifications.
-As you can see, this depends a lot on which kind of verifications you've enabled in your organization, so it's easy to say
-that most of the time an organization will not see any conflicts in this panel.
+For security, try to be extra cautious in these cases, and make sure an administrator reviews these potential conflicts.
+It depends a lot on which kind of verifications you've enabled in your organization, so most of the time an organization 
+don't see any conflicts appear in this panel.
 
-image:verifications_conflicts.png[Verification's conflicts panel]
+image:participants/verifications_conflicts.png[Verification's conflicts panel]
 
-== Transfer user
+When there is a verification conflict, administrators receive an email notification. 
 
-An administrator can review these conflicts by clicking in the Transfer user image:action_transfer_user.png[Transfer user] icon.
+== Manage verifications conflicts
 
-Then they will see the Transfer user form, where they can provide a reason for the transfer.
+In the verifications conflict panel, you can find all the conflicts, with:
+* *User*: name of the non managed participant who tried to get verified from their account.
+* *Managed user*: name of the impersonated user with the same verification data. 
+* *Attempts*: number of time the user tried to get verified with their data. 
+* *Solved*: if the verification conflict was solved or not. 
 
-image:verifications_conflicts_transfer_user.png[]
+You can search for conflicts by participant email, name, or nickname in the search bar. 
+
+=== Actions
+
+|===
+|Icon |Name |Definition
+
+|image:icons/action_transfer_user.png[Transfer user]
+|Transfer user
+|To solve the verification conflict.  
+
+|===
+
+=== Transfer user
+
+The transfer user button allows to solve the conflict between both account. 
+
+image:participants/verifications_conflicts_transfer_user.png[Transfer user form]
 
 .Transfer user form
 |===
 |Field |Type |Description
 
 |Reason
 |Required
-|Why you want to transfer the user
+|Why you want to transfer the user account
 
 |Email
 |Required
 |The email of the user to transfer
 
 |===
 
-After the user is transferred, both accounts (the original and the new one) will be merged.
+After the user is transferred, both accounts, the managed one and the new one, is merged into one, with the 
+email address entered in the form. 
```

Comment:
And another thing that happens is: That the verification is transferred to this account.

Reply:
Updated also!